app/scripts/lib/app-start.js

```diff
@@ -353,7 +353,8 @@ function (
     },
 
     initializeAuthenticationBroker: function () {
-      //jshint maxcomplexity: 7
+      /*jshint maxcomplexity: 7 */
+      /*eslint complexity: [2, 7] */
```

Comment:
inconsistent spaces, ` /*jshint maxcomplexity: 7 */` should there be a space after 7? 
`/*globals FxaHead*/` does not have spaces